pilot/docker/envoy_pilot.yaml.tmpl

```diff
@@ -135,6 +135,8 @@ static_resources:
               path: /dev/stdout
             name: envoy.file_access_log
           codec_type: HTTP2
+          http2_protocol_options:
+            max_concurrent_streams: 2147483646
```

Comment:
That's overshooting it a it, the maximum one side can send is 2^30, i.e. 1,073,741,824.

Reply:
Corrected. I was following the spec literally.